<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="backLayout" stretch="10,10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="10,10,10">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout1" stretch="1,5,0">
          <item>
           <widget class="QLabel" name="label1">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>CPU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit1"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox1">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout2">
          <item>
           <widget class="QLabel" name="label2">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string>GPU</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit2"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="price_label">
            <property name="text">
             <string>价格</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label22">
            <property name="text">
             <string>并行异构性价比：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="information31">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label11">
            <property name="text">
             <string>单线程串行指令数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="information12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="information21">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label32">
            <property name="text">
             <string>浮点性能性价比：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="information11">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>保存方案</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="information22">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label31">
            <property name="text">
             <string>浮点计算数FLOPS:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label21">
            <property name="text">
             <string>多线程扩展指令数</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label12">
            <property name="text">
             <string>串行性能性价比：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="information32">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label0">
            <property name="text">
             <string>店铺名称</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_store">
            <property name="text">
             <string>中文句子</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>comboBox1</receiver>
   <slot>setEditText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
